lib/puppet.rb

```diff
@@ -33,6 +33,10 @@
   def _(msg)
     msg
   end
+
+  def n_(msg_singular, msg_plural, count)
+    return count == 1 ? msg_singular : msg_plural
```

Comment:
This works as long as we only use a single plural form.. but strictly speaking, the n_ definition we're using from fast_getext supports more (and a block form, actually): https://github.com/grosser/fast_gettext/blob/master/lib/fast_gettext/translation.rb#L28-L43

It might be safer not to assume the number of args and index on count?